From ddaney@caviumnetworks.com  Wed Jun  3 16:56:04 2009
From: David Daney <ddaney@caviumnetworks.com>
Date: Tue,  5 May 2009 17:35:22 -0700
Subject: Staging: octeon: Hookup octeon-ethernet driver.
To: linux-mips@linux-mips.org, ralf@linux-mips.org, gregkh@suse.de
Cc: David Daney <ddaney@caviumnetworks.com>
Message-ID: <1241570122-22728-7-git-send-email-ddaney@caviumnetworks.com>


The previous patch adds the driver files for octeon-ethernet.  Here we
hook them up into the main kernel build system.

Signed-off-by: David Daney <ddaney@caviumnetworks.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/Kconfig  |    2 ++
 drivers/staging/Makefile |    1 +
 2 files changed, 3 insertions(+)

--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -125,5 +125,7 @@ source "drivers/staging/pata_rdc/Kconfig
 
 source "drivers/staging/udlfb/Kconfig"
 
+source "drivers/staging/octeon/Kconfig"
+
 endif # !STAGING_EXCLUDE_BUILD
 endif # STAGING
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -45,3 +45,4 @@ obj-$(CONFIG_VT6655)		+= vt6655/
 obj-$(CONFIG_USB_CPC)		+= cpc-usb/
 obj-$(CONFIG_RDC_17F3101X)	+= pata_rdc/
 obj-$(CONFIG_FB_UDL)		+= udlfb/
+obj-$(CONFIG_OCTEON_ETHERNET)	+= octeon/
